commit 6915193f653fd93c371cfc02164ff73c1e4ac82d
Author: Ben Skeggs <bskeggs@redhat.com>
Date:   Fri Mar 17 00:48:30 2017 +0000

    drm/nouveau/disp/nv50-: specify ctrl/user separately when constructing classes
    
    [ Upstream commit 2a32b9b1866a2ee9f01fbf2a48d99012f0120739 ]
    
    Signed-off-by: Ben Skeggs <bskeggs@redhat.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1442_6915193f653fd93c371cfc02164ff73c1e4ac82d.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1442_6915193f653fd93c371cfc02164ff73c1e4ac82d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
